### PR TITLE
Leaflet

### PR DESCRIPTION
**Pull request by hcwinsemius**
List and create views are prepared. This should be enough for a good user experience.

### CODE REVIEW

File: portal/static/map_create.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
+$(document).ready(function () {
+    pos_y = document.getElementById('position_y')
+    pos_x = document.getElementById('position_x')
+    crs = document.getElementById('position_crs')
+    pos_y.value = -6.8;
+    pos_x.value = 39.25;
+    var maxAutoZoom = 15;
+    var map = L.map('map').setView({lon: 0, lat: 0}, 2);
+    var osmLayer = L.tileLayer('https://osm.vtech.fr/hot/{z}/{x}/{y}.png?uuid=2fc148f4-7018-4fd0-ac34-6b626cdc97a1', {
```
Comment (RickHagenaars) [inline portal/static/map_create.js:9]: Is there a reason why we're using this tile server? Looks similar to the official OSM tiles to me?

File: portal/static/map_create.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        maxZoom: 18,
+        attribution: '&copy; <a href="https://openstreetmap.org/copyright" target="_blank">OpenStreetMap contributors</a> ' +
+            '| <a href="https://cloud.empreintedigitale.fr" target="_blank">Empreinte digitale</a>',
```
Comment (RickHagenaars) [inline portal/static/map_create.js:12]: Why is there a "empreintedigitale.fr" attribution?

File: portal/templates/site/create.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+{% block head %}
+    {{ super() }}
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css"
+   integrity="sha512-xodZBNTC5n17Xt2atTPuE1HxjVMSvLVW9ocqUKLsCC5CXdbqCmblAshOMAS6/keqq/sMZMZ19scR4PsZChSR7A=="
+   crossorigin=""/>
+
+{% endblock %}
+{% block body %}
+    {{ super() }}
+    <script src="https://unpkg.com/leaflet@1.7.1/dist/leaflet.js"
```
Comment (RickHagenaars) [inline portal/templates/site/create.html:12]: Can this be moved to the block tail?

File: portal/templates/site/list.html [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    <script src="https://unpkg.com/leaflet@1.7.1/dist/leaflet.js"
+        integrity="sha512-XQoYMqMTK8LvdxXYG3nZ448hOEQiglfqkJs1NOQV44cWnUrBc8PkAOcXy20w0vlaXaVUearIOBhiXZ5V3ynxwA=="
+        crossorigin="">
+    </script>
+    <script>
+        $(document).ready(function () {
+            // set maximum zoom level for auto zooming to the user available locs
+            var maxAutoZoom = 15;
+            var map = L.map('map').setView({lon: 0, lat: 0}, 2);
+            var osmLayer = L.tileLayer('https://osm.vtech.fr/hot/{z}/{x}/{y}.png?uuid=2fc148f4-7018-4fd0-ac34-6b626cdc97a1', {
```
Comment (RickHagenaars) [inline portal/templates/site/list.html:27]: Tile server same remark as above.

File: portal/templates/site/list.html [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+                maxZoom: 18,
+                attribution: '&copy; <a href="https://openstreetmap.org/copyright" target="_blank">OpenStreetMap contributors</a> ' +
```
Comment (RickHagenaars) [inline portal/templates/site/list.html:29]: Attribution same remark as above.

**Review by RickHagenaars — COMMENTED** [review]
Looks really good!

Some questions/remarks:
1. Is it intended that the edit page doesn't have the map and additional features? 
2. The EPSG code gets overwritten when you change coordinates. That would be an issue if want to allow users to configure a EPSG which deviates from what you auto-compute? If we always want to auto-compute I suggest we move that field under the lat/long and also implement the auto-compute on the edit page.
3. Pre-filling the EPSG/lng/lat when creating a new site does seem a bit confusing?

File: portal/static/map_create.js (hunk 1 above)
Comment (hcwinsemius) [reply portal/static/map_create.js:9]: have replaced this

File: portal/templates/site/list.html (hunk 4 above)
Comment (hcwinsemius) [reply portal/templates/site/list.html:27]: Changed

File: portal/templates/site/list.html (hunk 5 above)
Comment (hcwinsemius) [reply portal/templates/site/list.html:29]: removed

**Pull request merged**